test/dummy/config/environments/test.rb

```diff
@@ -30,6 +30,8 @@
   # ActionMailer::Base.deliveries array.
   # config.action_mailer.delivery_method = :test
 
+  config.active_support.test_order = :sorted
```

Comment:
Was there an issue with random test order?

Reply:
Oops. I typed this in wrong. Should be :random ! I'll push a fix.